data/templates/default.typst

```diff
@@ -31,12 +31,12 @@ $endif$
 $if(smart)$
 $else$
 #set smartquote(enabled: false)
-
 $endif$
+
```

Comment:
Why did you move the endif here?  Putting the space before the endif is a feature: it means that we'll only get the additional blank line if we get the `#set smartquote` line.  Putting it after will give us a blank line even if there is nothing.

Reply:
fair enough: reverted